### ISSUE TITLE
Debug crashed new 8 column run

### ISSUE THREAD

**Issue by raspstephan** [raspstephan/CBRAIN-Keras-Diagnostics#9]
NN experiment 001 crashed after around 10 days in CAM. Why? Previous experiments with 7 layers and those inputs and outputs worked, see PNAS paper.

To Debug:
---

- [x] Check why the run crashes. Is it typical or is there something obviously wrong.
- [x] Run new NN (006) with exactly the same setup as reference PNAS version (D025). Same layers, same input normalization. But note that I am testing with much fewer epochs.
- [x] If this crashes, plug in the weights and norm files from D025 to check whether the difference is in the neural network or in CAM

**Comment by raspstephan** [comment]
Surface PHQ and TPHYSTND blow up in extratropics and polar regions. Feel like we had that before. Doesn't seem like a bug but rather like a typical blow up. See c0f1e9a notebook 6.0 part 3

**Comment by raspstephan** [comment]
NN 006 does not crash, see notebook 5.3 for a comparison of skill (basically equal) and the Jacobians (unnormalized) which are definitely not equal! This suggests to me that the input normalization does matter! Because sensitivity tests in the PNAS paper show that fewer layers also work.

**Comment by raspstephan** [comment]
Cole this. Take away message: Input normalization seems to matter OR it's really a little bit of luck whether networks crash or not.

If it is input normalization: maxrs works, std_by_var doesn't. maxrs has much larger values for lower tropopause, gives less importance to upper levels.
